modules/search-service-impl/src/main/java/org/opencastproject/search/impl/solr/SolrIndexManager.java

```diff
@@ -693,14 +694,18 @@ static String mkString(Collection<?> as, String sep) {
 
   private Mpeg7Catalog loadMpeg7Catalog(Catalog cat) throws IOException {
     InputStream in = null;
+    File f = null;
     try {
-      File f = workspace.get(cat.getURI());
+      f = workspace.get(cat.getURI(), true);
       in = new FileInputStream(f);
       return mpeg7CatalogService.load(in);
```

Comment:
Since this wants an InputStream, we could do workspace.read(cat.getURI()) here

Reply:
@lkiesow, do you want to change that in this PR? There are now 2 downstream PRs that depend on this one, so would be good to get it merged.

Reply:
@smarquard Please answer @lkiesow 's question above so that we can proceed.

Reply:
Which question? Or did you get me and Lars the wrong way around here? I think I'm waiting for him.

Reply:
@smarquard Sorry, I was to fast in reading this ;-)

@lkiesow You owe Stephen an answer here.

Reply:
Sorry for the delay. I've added a patch to use `Workspace::read` instead of creating a temporary file in the workspace.